### PR TITLE
Creep rules

### PR DESCRIPTION
**Pull request by reverendbedford**
Adds all the accoutrements needed to implement the minimum creep rate models in the new draft of the EPP primary load code case.

### CODE REVIEW

File: doc/sphinx/creep/s_crmo.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+673.15      468
+723.15      452
+773.15      418
+823.15      634
+873.15      284
+894.15      300
+922.15      270
+=========== =====
+
+Because the model is fully parameterized in the C++ implement it must
```
Comment (arovinelli) [inline doc/sphinx/creep/s_crmo.rst]: implement -> implementation

File: test/test_creep.py [hunk 2]
```diff
@@ -225,6 +245,62 @@ def test_g(self):
...
+    self.t = 10.0
+
+    self.U = interpolate.PiecewiseLinearInterpolate(
+        [371, 400, 450, 500, 550, 600, 621, 649],
+        [471,468,452,418,634,284,300,270])
+
+  def test_g(self):
+    g_direct = self.model.g(self.s, self.e, self.t, self.T)
+    Uv = self.U.value(self.T - 273.15)
+    g_calc = 10.0**(11.498-8.2226*Uv/self.T-20448/self.T+5862.4/self.T*np.log10(self.s))/100.0
```
Comment (chakra34) [inline test/test_creep.py]: might help in clarity if there is a space between the individual terms

**Comment by chakra34** [comment]
It looks good to me

File: doc/sphinx/creep/s_crmo.rst (hunk 1 above)
Comment (reverendbedford) [reply doc/sphinx/creep/s_crmo.rst]: Yep, thanks.

File: test/test_creep.py (hunk 2 above)
Comment (reverendbedford) [reply test/test_creep.py]: Okay, done.

**Comment by reverendbedford** [comment]
@chakra34 @arovinelli  If you're good now one of you please approve.

**Pull request merged**